src/pages/3rd-party/index.hbs

```diff
@@ -97,6 +97,11 @@ layout: main.hbs
             <td>A GWT wrapper for OpenLayers 3+ written in Java</td>
             <td><a href="https://github.com/TDesjardins">Tino Desjardins</a></td>
           </tr>
+          <tr>
+            <td><a href="https://github.com/terrestris/react-geo">react-geo</a></td>
+            <td>A set of geo related modules to use in combination with react, antd and OpenLayers</td>
```

Comment:
Do you want to remove the "antd" word here?

Reply:
The antd word is actually a library…: https://github.com/ant-design/ant-design

Reply:
Its the name of the ui-package: https://www.npmjs.com/package/antd but maybe "Ant Design" would be better.

Reply:
PR welcome @KaiVolland

Reply:
That was confusing :)

Reply:
See #91